src/Backend/Modules/Quotes/info.xml

```diff
@@ -12,7 +12,7 @@
   </description>
   <authors>
     <author>
-      <name><![CDATA[Dylan Moerman]]></name>
```

Comment:
Erased from existance

Reply:
![who you talking about](https://i.imgflip.com/128gkv.jpg)